crates/ark/src/lsp/completions/completion_utils.rs

```diff
@@ -0,0 +1,123 @@
+use tree_sitter::Node;
+
+use crate::lsp::document_context::DocumentContext;
+use crate::lsp::traits::rope::RopeExt;
+use crate::treesitter::node_text;
+use crate::treesitter::NodeType;
+use crate::treesitter::NodeTypeExt;
+use crate::treesitter::UnaryOperatorType;
+
+#[allow(dead_code)]
```

Comment:
Several of these fields are actually used, but they were used along the way, i.e. when I had more logging in. I imagine this entire struct will go away when we do #681, so seems ok to have around for now.